spec/install/gemfile/path_spec.rb

```diff
@@ -418,6 +452,7 @@
     end
 
     it "gets dependencies that are updated in the path" do
+      build_lib "rack", "1.0.0", :path => lib_path("foo")
```

Comment:
This looks like a bad test that was somehow hidden by the dependency order causing a re-resolve.  We depend on rack, we're using a path in the Gemfile, and we weren't building rack in the path, so it can't be found.  I'm not sure how this passed previously, unless, like the above tests, the bug I'm fixing caused these bad tests to quietly pass.